examples/hn.svelte.dev/netlify.toml

```diff
@@ -1,10 +1,13 @@
 [build]
   command = "npm run build"
   publish = ".svelte-kit/netlify/build/"
-  functions = ".svelte-kit/netlify/functions/"
 
 [build.environment]
   NPM_FLAGS="--prefix=/dev/null"
 
 [[plugins]]
   package = "/.netlify/netlify-plugin-pnpm"
+
+[dev]
+# Disable svelte-kit dev so we can test functions
```

Comment:
how does this disable `svelte-kit dev`?

Reply:
I'm having a hard time understanding how gatsby is involved in this or how you'd use gatsby and sveltekit together (they seem like alternatives to one another). 

The changes in this file seem like maybe they should be just temporary changes in your own project for testing purposes and not something we'd want to check in

Reply:
Sorry, I mean the **Netlify** cli, not Gatsby!! This is what I get for replying to GitHub notifications when I'm on holiday. But you make a fair point. We can remove this but. I can update it next week when I'm back.

Reply:
I've removed this. Sorry for the confusion